wp-includes/class-wp-https-detection.php

```diff
@@ -69,7 +69,7 @@ public function init() {
 	 * Schedules a cron event to check for HTTPS support.
 	 */
 	public function schedule_cron() {
-		if ( ! wp_next_scheduled( self::CRON_HOOK ) && ! $this->is_currently_https() ) {
```

Comment:
<img width="1150" alt="ensure-secure-connection" src="https://user-images.githubusercontent.com/4063887/45376816-694f0480-b5be-11e8-9dde-a1a4acf7dceb.png">

Reply:
Maybe we should just have it always be “Force secure connections” regardless of whether HTTPS is on.

Reply:
That sounds good, as it will cover whether HTTPS is enabled or not.